README.md

```diff
@@ -1,5 +1,31 @@
 # Introduction 
-Project Coimbra
+Project Coimbra 

```

Comment:
this looks to be the same change as your other PR: https://github.com/EnableIrelandAT/Coimbra/pull/51/files ?

Reply:
Yes, please revert this as we should keep them as separate PRs.